realm/realm-library/src/main/java/io/realm/internal/OsResults.java

```diff
@@ -85,8 +85,6 @@ public T next() {
 
         /**
          * Not supported by Realm collection iterators.
-         *
-         * @throws UnsupportedOperationException
```

Comment:
Not sure I understand this? The implementation does throw?

Reply:
yeah, but empty description of `@throws` causes lint warnings. And the description is really redundant for this obvious deprecated method. So ...